app/ui/browser-modern/views/browser/decorations/statusbar.jsx

```diff
@@ -58,10 +60,16 @@ StatusBar.displayName = 'StatusBar';
 
 StatusBar.propTypes = {
   statusText: PropTypes.string,
```

Comment:
Since you're here anyway, might want to make `statusText` isRequired as well, since it's always passed in via props. AFAIK empty strings still registered as properly passed in props.